WordPress/Classes/ViewRelated/Me/MyDomains/DomainPurchaseChoicesView.swift

```diff
@@ -0,0 +1,170 @@
+             title: Strings.chooseSiteTitle,
+             subtitle: Strings.chooseSiteSubtitle,
+             buttonTitle: Strings.chooseSiteButtonTitle,
+             footer: Strings.chooseSiteFooter,
+             action: chooseSiteAction
+        )
+    }
+
+    private func card(
+        imageName: String,
+        title: String,
+        subtitle: String,
+        buttonTitle: String,
+        footer: String? = nil,
+        action: @escaping () -> Void
+    ) -> some View {
+        VStack(alignment: .leading, spacing: Length.Padding.single) {
+            Group {
+                Image(imageName)
+                    .renderingMode(.template)
+                    .resizable()
+                    .foregroundStyle(Color.DS.Background.brand)
+                    .frame(width: Constants.imageLength, height: Constants.imageLength)
+                    .padding(.top, Length.Padding.double)
+                VStack(alignment: .leading, spacing: Length.Padding.single) {
+                    Text(title)
+                        .font(.title2.bold())
+                    Text(subtitle)
+                        .foregroundStyle(Color.DS.Foreground.secondary)
+                    if let footer {
+                        Text(footer)
+                            .foregroundStyle(Color.DS.Foreground.brand)
+                    }
+                }
+                .padding(.bottom, Length.Padding.single)
+                PrimaryButton(title: buttonTitle) {
+                    action()
+                }
+                .padding(.bottom, Length.Padding.double)
+            }
+            .padding(.horizontal, Length.Padding.double)
+        }
+        .background(Color.DS.Background.primary)
+        .clipShape(RoundedRectangle(cornerRadius: 8))
+    }
+
+    private var chooseSiteTexts: some View {
+        VStack(alignment: .leading, spacing: Length.Padding.single) {
+            Text(Strings.chooseSiteTitle)
+                .font(.title2.bold())
+            Text(Strings.chooseSiteSubtitle)
+                .foregroundStyle(Color.DS.Foreground.secondary)
+            Text(Strings.chooseSiteFooter)
+                .foregroundStyle(Color.DS.Foreground.brand)
+        }
+    }
+}
+
+private extension DomainPurchaseChoicesView {
+    enum Strings {
+        static let header = NSLocalizedString(
+            "domain.management.purchase.title",
+            value: "Choose how to use your domain",
+            comment: "Domain management purchase domain screen title."
+        )
+        static let subheader = NSLocalizedString(
+            "domain.management.purchase.subtitle",
+            value: "Don't worry, you can easily add a site later.",
+            comment: "Domain management purchase domain screen title"
+        )
+
+        static let buyDomainTitle = NSLocalizedString(
+            "domain.management.buy.card.title",
+            value: "Just buy a domain",
+            comment: "Doman management buy domain card title"
```

Comment:
Typo:
```suggestion
            comment: "Domain management buy domain card title"
```
Same comment applies to the rest of the strings except the first two.